layers/+distributions/spacemacs-base/local/hybrid-mode/hybrid-mode.el

```diff
@@ -118,12 +118,16 @@
   :exit-hook (evil-cleanup-insert-state evil-stop-track-last-insertion)
   :input-method t
   (cond
-   ((evil-hybrid-state-p)
+   ((evil-insert-state-p)
```

Comment:
Thanks @nixmaniack, it was a careless mistake on my part. Fixed and rebased, should be `evil-hybrid-state-p` now.